test/e2e/instrumentation-hook/register-once/register-once.test.ts

```diff
@@ -0,0 +1,17 @@
+import { nextTestSetup } from 'e2e-utils'
+
+describe('instrumentation-hook - register-once', () => {
+  const { next, skipped } = nextTestSetup({
+    files: __dirname,
+    skipDeployment: true,
+  })
+
+  if (skipped) {
+    return
+  }
+
+  it('should only register once', async () => {
+    await next.fetch('/foo')
+    expect(next.cliOutput.split('register-log').length).toBe(2)
```

Comment:
When expecting a certain number of log occurrences, I find the split/length approach always hard to read. It also doesn't give a nice expected/received output in case of errors. I would suggest this instead:

```suggestion
    expect(next.cliOutput).toIncludeRepeated('register-log', 1)
```